### PR TITLE
OMC poll for January

### PR DESCRIPTION
**Pull request by LongForWisdom**

### CODE REVIEW

File: governance/polls/PPG - Open Market Committee Proposal - January 17, 2022.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+**Stability Fee Decreases**
+* Decrease the ETH-A [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) from 2.75% to 2.5%.
+* Decrease the WSTETH-A [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) from 4% to 3%.
+
+**Debt Ceiling Decreases**
+* Decrease the LINK-A [Maximum Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) from 140 million DAI to 100 million DAI.
+* Decrease the YFI-A [Maximum Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) from 130 million DAI to 50 million DAI.
+* Decrease the UNI-A [Maximum Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) from 50 million DAI to 25 million DAI.
+* Decrease the UNIV2UNIETH-A [Maximum Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) from 20 million DAI to 5 million DAI.
+* Decrease the GUSD-A [Debt Ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) from 5 million DAI to zero.
```
Comment (ultraschuppi) [inline governance/polls/PPG - Open Market Committee Proposal - January 17, 2022.md:38]: ... to zero DAI?

File: governance/polls/PPG - Open Market Committee Proposal - January 17, 2022.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+**DIRECT-AAVEV2-DAI (Aave D3M) Changes**
+* Decrease the DIRECT-AAVEV2-DAI Target Borrow Rate from 3.9% to 3.75%.
```
Comment (ultraschuppi) [inline governance/polls/PPG - Open Market Committee Proposal - January 17, 2022.md:41]: maybe link to [bar](https://github.com/makerdao/dss-direct-deposit#configuration)

**Review by ultraschuppi — COMMENTED** [review]
just minor suggestions, numbers are right

**Review by ultraschuppi — APPROVED** [review]

File: governance/polls/PPG - Open Market Committee Proposal - January 17, 2022.md (hunk 1 above)
Comment (LongForWisdom) [reply governance/polls/PPG - Open Market Committee Proposal - January 17, 2022.md:38]: Yep

File: governance/polls/PPG - Open Market Committee Proposal - January 17, 2022.md (hunk 2 above)
Comment (LongForWisdom) [reply governance/polls/PPG - Open Market Committee Proposal - January 17, 2022.md:41]: Yeah, better than nothing.

**Pull request merged**